R/raster_by_poly.R

```diff
@@ -15,6 +15,11 @@
 #' @export
 raster_by_poly <- function(raster_layer, poly, poly_field, summarize = FALSE,
                            parallel = FALSE) {
+  # I'm not deprecating this right now as both raster and sp are not
```

Comment:
Do you think this comment might be better put into the function documentation?

Reply:
I don't think so... if it's not marked for deprecation then I would say it's clear enough it won't be. Just a note to ourselves when we go to make them defunct